zshot/linker/linker_smxm.py

```diff
@@ -12,19 +12,20 @@
 )
 from zshot.utils.data_models import Span
 
-MODEL_NAME = "ibm/smxm"
+ONTONOTES_MODEL_NAME = "ibm/smxm"
 
 
 class LinkerSMXM(Linker):
     """ SMXM linker """
 
-    def __init__(self):
+    def __init__(self, model_name=ONTONOTES_MODEL_NAME):
         super().__init__()
 
         self.tokenizer = BertTokenizerFast.from_pretrained(
-            "bert-large-cased", truncation_side="left"
+            "bert-large-cased" if model_name == ONTONOTES_MODEL_NAME else model_name, truncation_side="left"
```

Comment:
tokenizer should probably be `bert-large-cased` also in the case of med-mentions and custom models